isodatetime/data.py

```diff
@@ -1720,10 +1720,14 @@ def _get_dump_format(self):
 
         if self.get_is_calendar_date():
             date_string = year_string + "-MM-DD"
-        if self.get_is_ordinal_date():
+        elif self.get_is_ordinal_date():
             date_string = year_string + "-DDD"
-        if self.get_is_week_date():
+        elif self.get_is_week_date():
             date_string = year_string + "-Www-D"
+        else:
+            raise RuntimeError("Inconsistent state before dumping. Must"
```

Comment:
Perhaps:

> TimePoint has inconsistent state, points must conform to calendar, ordinal or week dates.

Reply:
+1 So much better! Changing right now. Thanks!

Reply:
Done, also rebased branch onto master. Travis should - hopefully - build it with no issues :crossed_fingers: